server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSetMethodContract.scala

```diff
@@ -538,13 +531,8 @@ trait EmailSetMethodContract {
       .asString
 
     assertThatJson(response)
-      .inPath("methodResponses[1][1].list[0]")
-      .isEqualTo(String.format(
-        """{
-          |   "id":"%s",
-          |   "keywords": {}
-          |}
-      """.stripMargin, messageId.serialize))
+      .inPath("methodResponses[0][1].notUpdated")
+      .isEqualTo(s"""{"${messageId.serialize}":{\"type\":"serverFail\",\"description\":\"#private:andre@domain.tld:andrecustom can not be found\"}}""")
```

Comment:
Indent :-)

& serverFail => notFound

& You disclose mailbox name. Maybe it is better to not put description at all.